readme.md

```diff
@@ -152,6 +152,7 @@
 - [Fortran](https://github.com/rabbiabram/awesome-fortran)
 - [.NET](https://github.com/quozd/awesome-dotnet)
 	- [Core](https://github.com/thangchung/awesome-dotnet-core)
+	- [Roslyn](https://github.com/ironcev/awesome-roslyn)
```

Comment:
There was rule that the URL should end with `#readme`. Not sure if it still applies.

Reply:
Yes, should end in `#readme`

Reply:
> The entry in the Awesome list should:
> Include a short description about the project/theme of the list. **It should not describe the list itself.**